source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -299,15 +400,36 @@ def _get_matrix(cls, json_response, origins, destinations):
 
         return sn_routing_matrix
 
-    def get_matrix_params(self, origins, destinations, mode, max_duration, request):
+    @classmethod
+    def _create_matrix_response_with_direct_path(cls, json_response, origin, destination, row):
```

Comment:
this function is not a classmethod but a staticmethod

Reply:
`origin, destination` are not used

Reply:
oops this is an evil copy/paste